docs/developer-guide/working-with-rules.md

```diff
@@ -198,9 +198,24 @@ The `fixer` object has the following methods:
 
 Best practices for fixes:
 
-1. Make fixes that are as small as possible. Anything more than a single character is risky and could prevent other, simpler fixes from being made.
+1. Avoid any fixes that could change the runtime behavior of code and cause it to stop working.
+1. Make fixes as small as possible. Fixes that are unnecessarily large could conflict with fixes from other rules, and prevent them from being applied.
```

Comment:
Technically, even fixes from the same rule (if they have overlapping ranges) could conflict. I wonder if we should just say "could conflict with other fixes"?